Sources/Views/PricingLanding.swift

```diff
@@ -352,70 +355,75 @@ private func pricingPlan(_ plan: PricingPlan) -> ChildOf<Tag.Ul> {
 }
 
 private let whatToExpect = [
-  gridRow(
-    [
-      `class`([
-        Class.padding([.mobile: [.leftRight: 2, .topBottom: 3], .desktop: [.all: 4]])
-        ]),
-      style(backgroundColor(.other("#fafafa")))
-    ],
+  div(
```

Comment:
this file has a lot of churn due to some indentation, so may be best to review with "ignore whitespace" on